collectors/diskspace.plugin/plugin_diskspace.c

```diff
@@ -687,7 +665,7 @@ void *diskspace_main(void *ptr) {
                 continue;
 
             // exclude mounts made by ProtectHome and ProtectSystem systemd hardening options
-            if(mi->flags & MOUNTINFO_READONLY && !strcmp(mi->root, mi->mount_point))
+            if(mi->flags & MOUNTINFO_READONLY && mi->flags & MOUNTINFO_IS_PROTECTED && !strcmp(mi->root, mi->mount_point))
```

Comment:
isn't `mi->flags & MOUNTINFO_IS_PROTECTED` is enough here after the change?

Reply:
Not by itself...

But it can if we move the other checks too **when** setting the flag.

As I understand it, we deduce basically that it is protected if it matches it's root, and it is read only and is in the list of protected by systemd.

Reply:
Perhaps rename the flag to `MOUNTINFO_IS_IN_SYSTEMD_PROTECTED_LIST` or something?

Reply:
Done!